src/scores/probability/crps_impl.py

```diff
@@ -825,6 +826,8 @@ def crps_for_ensemble(
 
     See also:
         :py:func:`scores.probability.crps_cdf`
+        :py:func:`twcrps_for_ensemble`
+        :py:func:`tail_twcrps_for_ensemble`
```

Comment:
```suggestion
        :py:func:`scores.probability.twcrps_for_ensemble`
        :py:func:`scores.probability.tail_twcrps_for_ensemble`
```
Is it better to include "scores.probability." at the start of both of these (as per `scores.probability.crps_cdf`)? I honestly don't know which is preferable, but I wanted to ask.

Reply:
Good pickup - I'll fix this